spec/models/payroll_run_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe PayrollRun, type: :model do
+  describe "#total_award_amount" do
+    it "returns the sum of the award amounts of its claims" do
+      claim_1 = build(:claim, :approved, eligibility: build(:student_loans_eligibility, :eligible, student_loan_repayment_amount: 1500))
+      claim_2 = build(:claim, :approved, eligibility: build(:student_loans_eligibility, :eligible, student_loan_repayment_amount: 2000))
+
+      payroll_run = PayrollRun.new(claims: [claim_1, claim_2])
+
+      expect(payroll_run.total_award_amount).to eq(3500)
+    end
+  end
+
+  describe ".payrollable_claims" do
```

Comment:
We deliberated between a class method on `PayrollRun` and a scope on `Claim`. The only reason we decided to put it here is because we worried that, since every feature in the app has _something_ to do with claims, it would be quite easy for `Claim` becoming a dumping ground for a lot of stuff.